### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by pjenys0**
- using filepath.Base/filepath.Join to get command name + full file path
- Improve CreateFile() functions.
- Improve logging

### CODE REVIEW

File: cmd/install.go [hunk 1]
```diff
@@ -527,15 +529,16 @@ func setupBasePasteWorkingEnvironment(ctx context.Context, config *configs.Confi
 	config.RPCPwd = utils.GenerateRandomString(15)
 
 	// create pastel.conf file
-	f, err := utils.CreateFile(ctx, config.WorkingDir+"/pastel.conf", config.Force)
+	pastelConfigPath := filepath.Join(config.WorkingDir, "pastel.conf")
```
Comment (georgkcode0) [inline cmd/install.go:532]: better if we replace `pastel.conf` to ` constants.PastelConfName`

**Review by georgkcode0 — APPROVED** [review]
LGTM,

File: utils/utils.go [hunk 2]
```diff
@@ -49,29 +49,29 @@ func CreateFolder(ctx context.Context, path string, force bool) error {
...
 
 	if force {
-		return fileName, create(fileName)
+		return create(filePath)
 	}
 
 	// create if not already exists
-	if _, err := os.Stat(fileName); os.IsNotExist(err) {
-		return fileName, create(fileName)
+	if _, err := os.Stat(filePath); os.IsNotExist(err) {
```
Comment (a-ok123) [inline utils/utils.go:70]: this if can be combined with `if force`

**Review by a-ok123 — APPROVED** [review]

File: utils/utils.go (hunk 2 above)
Comment (pjenys0) [reply utils/utils.go:70]: Thanks, will improve later in another PR. I also saw some not clean-logic in this function.

**Pull request merged**